cmd/web/routes.go

```diff
@@ -34,14 +35,16 @@ func (app *application) subscribe(w http.ResponseWriter, r *http.Request) {
 	}
 
 	newEmail := r.PostForm.Get("email")
-	if !isCorrectEmail(newEmail) {
-		app.clientError(w, http.StatusUnprocessableEntity)
+	v := validator.New()
```

Comment:
Using initiate validator inside of endpoint is not optimal cause it will be created each time this method is called. In this case, it's each request to this endpoint.
Do you consider moving it to application level?

Reply:
Clear, seems reasonable to me 👍